django/applications/catmaid/control/annotationadmin.py

```diff
@@ -299,7 +301,7 @@ def copy_annotations(source_pid, target_pid, import_treenodes=True,
                 JOIN class_instance ci_s ON ci_s.id=cici.class_instance_b
                 WHERE cici.project_id=%s AND relation_id=%s
                 ''')
-        except DoesNotExist:
+        except DoesNotExist: # FIXME: This exception is not defined, although Django has a number of more specific variants
```

Comment:
Right, glancing over the code, it should probably be `except Class.DoesNotExist, Relation.DoesNotExist:` and the code above should only execute the `INSERT` SQL if `annotations_src` has elements. But given that the SQL won't work anyway, and hence this code is not used actively, it might not be worth the effort. I will check in more detail if the whole `annotationadmin.py` functionality can be removed. Exports and imports are currently done through the CLI tools `catmaid_export_data` and `catmaid_import_data`.